### PR TITLE
Switch default branch from master to main

### PR DESCRIPTION
**Pull request by dhimmel**
work in progress

refs https://github.com/manubot/rootstock/issues/349

Switch default branch from master to main, while maintaining support for existing manuscripts that continue using master branch.

TODO:

- [x] update setup docs to explain change https://github.com/manubot/manubot/pull/275
- [x] update manubot/manubot tooling that assumes master is default

### CODE REVIEW

File: ci/deploy.sh [hunk 1]
```diff
@@ -12,7 +12,8 @@ REPO_SLUG=${TRAVIS_REPO_SLUG:-$GITHUB_REPOSITORY}
 COMMIT=${TRAVIS_COMMIT:-$GITHUB_SHA}
 CI_BUILD_WEB_URL=${CI_BUILD_WEB_URL:-$TRAVIS_BUILD_WEB_URL}
 CI_JOB_WEB_URL=${CI_JOB_WEB_URL:-$TRAVIS_JOB_WEB_URL}
-BRANCH=${TRAVIS_BRANCH:-master}
+BRANCH=${TRAVIS_BRANCH:-$DEFAULT_BRANCH}
+BRANCH=${BRANCH:-$main}
```
Comment (agitter) [inline ci/deploy.sh:16]: The approach looks good so far.  I can review when it's ready.

Did you want this here?
```suggestion
BRANCH=${BRANCH:-main}
```

Comment (dhimmel) [reply ci/deploy.sh]: Nice catch. Feel free to review now. Will probably add something to the setup update instructions for updating users that still have a master default. They do not need to change, but the documentation will make more sense this way.

File: .travis.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ services:
   - docker
 branches:
   only:
+    - main
```
Comment (agitter) [inline .travis.yml:8]: This legacy workflow could run on `main` but will only deploy when run on `master`. That's potentially inconsistent, but I don't think it is a problem.  As you note, this is a legacy workflow.

**Review by agitter — APPROVED** [review]
Looks good to me.  Adding a note to setup before merging is a good idea.

**Pull request merged**